packages/ts/react-i18n/src/index.ts

```diff
@@ -41,11 +46,19 @@ export class I18n {
       return;
     }
 
-    const newTranslations = await this.loadTranslations(newLanguage);
+    let translationsResult: TranslationsResult;
+    try {
+      translationsResult = await this.#backend.loadTranslations(newLanguage);
+    } catch (e) {
+      console.error(`Failed to load translations for language: ${newLanguage}`, e);
```

Comment:
Can we let errors go up into application code as discussed yesterday?

Reply:
We can, but doing that currently seems to break the app. We would need to handle it somewhere.

Reply:
Let's go with logging the error for now.